src/pixel/mod.rs

```diff
@@ -10,6 +10,15 @@ pub use self::gamma_rgb::GammaRgb;
 mod srgb;
 mod gamma_rgb;
 
+///A transfer function to and from linear space.
+pub trait TransferFn {
```

Comment:
It will be good to add rgb_to_linear and rgb_from linear with default implementation, one the linear and gamma rgb are defined.